File: monitor/events/list-get-example-date-filter/list-get-example-date-filter.5.x.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";

$client = new Client($sid, $token);

$events = $client->monitor->events()->read(
    array(
        'startDate' => '2015-03-01T00:00:00Z',
        'endDate' => '2015-04-01T00:00:00Z'
    )
);

foreach ($events as $event) {
    echo $event->description;
}

File: monitor/events/list-get-example-date-filter/list-get-example-date-filter.5.x.rb
require 'twilio-ruby'

# put your own credentials here
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']

# set up a client to talk to the Twilio REST API
@client = Twilio::REST::Client.new(account_sid, auth_token)

@params = {
  start_date: Time.new('2015-03-01T00:00:00Z'),
  end_date: Time.new('2015-04-01T00:00:00Z')
}

@client.monitor.v1.events.list(@params).each do |e|
  puts e.description
end
